voice/transfer-call-inline-ncco.py

```diff
@@ -1,40 +1,42 @@
-# Import dependencies
-import nexmo
-import os
+#Import the dependencies
+import nexmo, os
 from os.path import join, dirname
 from dotenv import load_dotenv
+from pprint import pprint
 
-# Load the environment
-envpath = join(dirname(__file__), './.env')
+#Load the environment
+envpath = join(dirname(__file__),'./.env')
 load_dotenv(envpath)
 
-# Init the client
+#Init the client
 print(os.getenv("NEXMO_PRIVATE_KEY"))
 client = nexmo.Client(
-    application_id=os.getenv('NEXMO_APPLICATION_ID'),
-    private_key=os.getenv("NEXMO_PRIVATE_KEY")
+    application_id = os.getenv('NEXMO_APPLICATION_ID'),
+    private_key = os.getenv("NEXMO_PRIVATE_KEY")
 )
 
 response = client.create_call({
-    "to": [{"type": "phone", "number": os.getenv('TO_NUMBER')}],
-    "from": {"type": "phone", "number": os.getenv('FROM_NUMBER')},
-    "ncco": [
-        {
-            "action": "talk",
-            "text": "This is just a text whilst you tranfer to another NCCO"
-        }
-    ]
+  'to': [{'type': 'phone', 'number': os.getenv('TO_NUMBER' }],
```

Comment:
`os.getenv()` is missing its closing parenthesis here (also the one on the next line!)